modules/op-difference-between-non-versioned-and-versioned-cluster-tasks.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is part of the following assembly:
+//
+// *cicd/pipelines/managing-nonversioned-and-versioned-cluster-tasks.adoc
+:_content-type: CONCEPT
+[id="difference-between-non-versioned-and-versioned-cluster-tasks_{context}"]
+= Difference between non-versioned and versioned cluster tasks
+
```

Comment:
- All modules and assemblies require a summary phrase that functions as an abstract. 
- Have you considered moving this table to modules/op-advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks.adoc ?